setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup
+
+requirements = [
+      'gym',
+      'chainer',
+      'numpy',
+      'Pillow']
+
+setup(name='text_localization_environment',
+      version='0.0.1',
+      install_requires=requirements  # And any other dependencies foo needs
```

Comment:
```suggestion
      install_requires=requirements
```

(The comment looks like a relict from somewhere you took this setup.py from 😉)